processor/src/stack/trace.rs

```diff
@@ -21,10 +21,10 @@ pub struct StackTrace {
 impl StackTrace {
     // CONSTRUCTOR
     // --------------------------------------------------------------------------------------------
-    /// Returns a [StackTrace] instantiated with empty vectors for all columns.
-    pub fn new(inputs: &ProgramInputs, init_trace_capacity: usize) -> Self {
+    /// Returns a [StackTrace] instantiated with the provided input values. When fewer than
+    /// `MIN_STACK_DEPTH` inputs are provided, the rest of the stack top elements are set to ZERO.
+    pub fn new(init_values: &[Felt], init_trace_capacity: usize) -> Self {
```

Comment:
I think we need to make a couple of updates to the construction:

1. Column $b_0$ should be initialized to the actual stack depth. It seems like right now it is always initialized to $16$. We should probably also add a test to catch a situation where the actual stack depth does not match what is recorded in $b_0$.
2. Column $b_1$ should be initialized to the address of the first row in the overflow table. Previously, it didn't need to be done since the overflow table was always empty and the value in this column was initialized to zero.

Reply:
Great point, thank you!